src/Component/Tasks/Tasks.jsx

```diff
@@ -7,112 +24,46 @@ const Tasks = () => {
         </div>
         <div className=" flex  flex-wrap justify-start items-center">
           <span className="mr-2">All Tags :</span>
-          <button className="mr-3 bg-gray-900 text-white w-20 h-7 rounded-2xl my-3">
-            work
-          </button>
-          <button className="mr-3 bg-gray-900 text-white w-20 h-7 rounded-2xl my-3">
-            work
-          </button>
+          {tag.map((tags) => {
+            return (
+              <button
+                key={tags}
+                className="mr-3 bg-gray-900 text-white px-3 py-1 rounded-2xl my-3"
+              >
+                {tags}
```

Comment:
I think the naming conventions should be `tags` on you are mapping and `tag` in the callback.

Reply:
yes modified it.